lib/compat/wordpress-6.1/template-parts-screen.php

```diff
@@ -0,0 +1,209 @@
+	$custom_settings      = array(
+		'siteUrl'                   => site_url(),
+		'postsPerPage'              => get_option( 'posts_per_page' ),
+		'styles'                    => get_block_editor_theme_styles(),
+		'defaultTemplateTypes'      => $indexed_template_types,
+		'defaultTemplatePartAreas'  => get_allowed_block_template_part_areas(),
+		'supportsLayout'            => WP_Theme_JSON_Resolver::theme_has_support(),
+		'supportsTemplatePartsMode' => current_theme_supports( 'block-template-parts' ),
+		'__unstableHomeTemplate'    => gutenberg_resolve_home_template(),
+	);
+
+	/**
+	 * We don't need home template resolution when block template parts are supported.
+	 * Set the value to true to satisfy the editor initialization guard clause.
+	 */
+	if ( $custom_settings['supportsTemplatePartsMode'] ) {
+		$custom_settings['__unstableHomeTemplate'] = true;
+	}
+
+	// Add additional back-compat patterns registered by `current_screen` et al.
+	$custom_settings['__experimentalAdditionalBlockPatterns']          = WP_Block_Patterns_Registry::get_instance()->get_all_registered( true );
+	$custom_settings['__experimentalAdditionalBlockPatternCategories'] = WP_Block_Pattern_Categories_Registry::get_instance()->get_all_registered( true );
+
+	$editor_settings = get_block_editor_settings( $custom_settings, $block_editor_context );
+
+	if ( isset( $_GET['postType'] ) && ! isset( $_GET['postId'] ) ) {
+		$post_type = get_post_type_object( $_GET['postType'] );
+		if ( ! $post_type ) {
+			wp_die( __( 'Invalid post type.', 'gutenberg' ) );
+		}
+	}
+
+	$active_global_styles_id = WP_Theme_JSON_Resolver::get_user_global_styles_post_id();
+	$active_theme            = wp_get_theme()->get_stylesheet();
+	$preload_paths           = array(
+		array( '/wp/v2/media', 'OPTIONS' ),
+		'/wp/v2/types?context=view',
+		'/wp/v2/types/wp_template?context=edit',
+		'/wp/v2/types/wp_template-part?context=edit',
+		'/wp/v2/templates?context=edit&per_page=-1',
+		'/wp/v2/template-parts?context=edit&per_page=-1',
+		'/wp/v2/themes?context=edit&status=active',
+		'/wp/v2/global-styles/' . $active_global_styles_id . '?context=edit',
+		'/wp/v2/global-styles/' . $active_global_styles_id,
+		'/wp/v2/global-styles/themes/' . $active_theme,
+	);
+
+	block_editor_rest_api_preload( $preload_paths, $block_editor_context );
+
+	wp_add_inline_script(
+		'wp-edit-site',
+		sprintf(
+			'wp.domReady( function() {
+				wp.editSite.initializeEditor( "site-editor", %s );
+			} );',
+			wp_json_encode( $editor_settings )
+		)
+	);
+
+	// Preload server-registered block schemas.
+	wp_add_inline_script(
+		'wp-blocks',
+		'wp.blocks.unstable__bootstrapServerSideBlockDefinitions(' . wp_json_encode( get_block_editor_server_block_settings() ) . ');'
+	);
+
+	wp_add_inline_script(
+		'wp-blocks',
+		sprintf( 'wp.blocks.setCategories( %s );', wp_json_encode( get_block_categories( $block_editor_context ) ) ),
+		'after'
```

Comment:
This is the default value, we don't need it here.

```suggestion
```

Reply:
I copied this from `site-editor.php,` so we might want to check the editor pages in core and remove them.